NMS-14337: Update Business Service Monitoring docs (#4763)
OpenNMS/opennms: docs/modules/operation/pages/bsm/introduction.adoc

= Business Service Monitoring

While {page-component-title} detects issues in your network by device, interface or service, the Business Service Monitoring (BSM) takes it one step further.

> With the BSM feature, it is possible to model high level business service contexts around the technical Service Monitors provided in {page-component-title} and quickly identify the most critical problems affecting them.

Reviewer: ```suggestion
With the BSM feature, it is possible to model high-level business service contexts around the technical Service Monitors provided in {page-component-title} and quickly identify the most critical problems affecting them.
```

> To indicate which business service is affected by events at the technical Service Monitors level, a business service Operational Status is calculated.

Reviewer: ```suggestion
To indicate which business service is affected by events at the technical service monitors level, a business service operational status is calculated.
```

As an example, let's assume a company runs an online store.
Customers log in, select items, place them in the shopping cart and checkout using a payment system.
The whole service is provided by a few web servers and access data from databases.
To monitor the status of the databases, a SQL service monitor on each database server is configured.
For testing the web servers a HTTP service monitor is used for each of them.

> Covering the overall functionality, a Page Sequence Monitor (PSM) is used to test the login, shop and payment workflow through the provided web portal.

Reviewer: ```suggestion
Covering the overall functionality, a Page Sequence Monitor (PSM) tests the login, shop, and payment workflow through the provided web portal.
```
is it the login workflow, shop workflow and payment workflow, or the "login and shop and payment workflow"?
If the former we can just add a comma and pluralize workflow: "login, shop, and payment workflows"

To be able to model this scenarios the BSM functions can be used.
The Business Service Monitoring (BSM) feature includes the following components:

> * Business Service Monitoring Daemon (BSMD): Maintains and drives the state of all business service

Reviewer: ```suggestion
* Business Service Monitoring Daemon (BSMD): Maintains and drives the state of all business services
```

> * Business Service Editor: Web application that lets you create, update, or delete business service

Reviewer: ```suggestion
* Business Service Editor: Web application that lets you create, update, or delete business services
```

* Topology View for Business Services: Visual representation of the Business Service Hierarchy as a component of the Topology User Interface.

> * BSM ReST API: ReST based API to create, read, update or delete business service

Reviewer: ```suggestion
* BSM REST API: REST-based API to create, read, update, or delete business services
```
Not sure if this should be plural here and in lines 24&25
